modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/service/impl/SXPElementLocalServiceImpl.java

```diff
@@ -120,6 +121,10 @@ public SXPElement deleteSXPElement(long sxpElementId)
 	public SXPElement deleteSXPElement(SXPElement sxpElement)
 		throws PortalException {
 
+		if (sxpElement.isReadOnly()) {
```

Comment:
I'm not sure this should be here: it should still be able to delete any element using the local service. It would be better to keep it in remote service.